doc/md/guides/overview/modern-database-ci-cd.mdx

```diff
@@ -0,0 +1,233 @@
+---
+title: Modern Database CI/CD with Atlas
+id: modern-database-ci-cd
+slug: modern-database-ci-cd
+---
+
+![](https://atlasgo.io/uploads/ci-cd-guide/database-ci-cd-workflow.png)
+
+Continuous integration and continuous delivery (CI/CD) are tenets of modern software development.
+CI/CD enables teams to deliver software faster, with higher quality, and with less risk. However,
+CI/CD is not just for application code.  Teams can also apply CI/CD principles to their databases,
+to extend the benefits of CI/CD to one of the most critical components of their stack.
+
+This document describes the recommended workflow for teams using Atlas to achieve continuous integration
+and continuous delivery (CI/CD) for their databases.
```

Comment:
Lot's of `CI/CD`. I think you can drop some of them.

Reply:
Agree.